### PR TITLE
Jacks branch

### PR DESCRIPTION
**Pull request by zhanjack822**
Added boilerplate code from Visual Studio for an asp.net core project

### CODE REVIEW

File: Source/Pages/Error.cshtml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿@page
+@model ErrorModel
+@{
+    ViewData["Title"] = "Error";
+}
+
```
Comment (JoshKazi) [inline Source/Pages/Error.cshtml:6]: I don't fw this tbh

Comment (zhanjack822) [reply Source/Pages/Error.cshtml:6]: Bruh this is literally boilerplate code.

Note: This is just to familiarize Josh with PRs

Comment (JoshKazi) [reply Source/Pages/Error.cshtml:6]: Aight bet crodie

File: Source/Pages/Error.cshtml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+<h1 class="text-danger">Error.</h1>
+<h2 class="text-danger">An error occurred while processing your request.</h2>
+
```
Comment (JoshKazi) [inline Source/Pages/Error.cshtml:9]: I don't fw this either

Comment (zhanjack822) [reply Source/Pages/Error.cshtml:9]: Just testing out single comment vs begin review

**Review by JoshKazi — APPROVED** [review]

**Pull request merged**